documentation/sdkautomation/sdk_customization.md

```diff
@@ -205,6 +126,10 @@ The working folder of all the scripts is the __root folder of sdk repo__.
   },
   "initOptions": {
     "initScript": {
+      // Script to init dependencies.
+      // Param: <path_to_initInput.json> <path_to_initOutput.json>
```

Comment:
It isn't clear to me what I'd do with these json files. Can you provide some example?

Reply:
initInput is now an empty file. initOutput has definition in the bottom of this file.